packages/web-components/playwright.config.ts

```diff
@@ -13,6 +13,23 @@ const config: PlaywrightTestConfig = {
       width: 1280,
     },
   },
+  projects: [
+    {
+      name: 'chromium',
+      use: { ...devices['Desktop Chrome'] },
+      testMatch: /.*\.spec\.ts$/,
+    },
+    {
+      name: 'firefox',
+      use: { ...devices['Desktop Firefox'] },
+      testMatch: [/set-theme\.spec\.ts$/],
+    },
+    {
+      name: 'webkit',
+      use: { ...devices['Desktop Safari'] },
+      testMatch: [/set-theme\.spec\.ts$/],
+    },
+  ],
```

Comment:
Yes, this should be running both locally and on CI. I didn't enable for all component tests because, after chatting with John, some components' tests need fixes to be able to run in other browsers.